src/olympia/reviewers/tests/test_views.py

```diff
@@ -6086,6 +6086,38 @@ def test_non_existent_requested_file_returns_404(self):
         response = self.client.get(self.url + '?file=UNKNOWN_FILE')
         assert response.status_code == 404
 
+    def test_requested_file_contains_whitespace(self):
+        new_version = version_factory(
+            addon=self.addon, file_kw={'filename': 'webextension_no_id.xpi'})
+
+        repo = AddonGitRepository.extract_and_commit_from_version(new_version)
+
+        apply_changes(
+            repo, new_version, u'(function() {})\n', u'content script.js')
```

Comment:
`u'...'` 
![gpg3k75](https://user-images.githubusercontent.com/768592/75246274-5ab2d800-57c7-11ea-8077-2d9b730a7c7b.jpg)

Reply:
Damn you copy & paste :see_no_evil:

Reply:
c025ed4